lectures/money_inflation.md

```diff
@@ -393,7 +388,7 @@ $$ (eq:method1)
 **Remark 1:** method 1 uses an indirect approach to computing an equilibrium by first computing an equilibrium  $\{R_t, b_t\}_{t=0}^\infty$ sequence and then using it to back out an equilibrium  $\{p_t, m_t\}_{t=0}^\infty$  sequence.
 
 
-**Remark 2:** notice that  method 1 starts by picking an **initial condition** $R_0$ from a set $[\frac{\gamma_2}{\gamma_1}, R_u]$. An equilibrium $\{p_t, m_t\}_{t=0}^\infty$ sequences are not unique.  There is actually a continuum of equilibria indexed by a choice of $R_0$ from the set $[\frac{\gamma_2}{\gamma_1}, R_u]$. 
+**Remark 2:** notice that  method 1 starts by picking an **initial condition** $R_0$ from a set $[\frac{\gamma_2}{\gamma_1}, R_u]$. Equilibrium $\{p_t, m_t\}_{t=0}^\infty$ sequences are not unique.  There is actually a continuum of equilibria indexed by a choice of $R_0$ from the set $[\frac{\gamma_2}{\gamma_1}, R_u]$. 
```

Comment:
@mmcky , is there a suitable environment for these remarks?

Reply:
@jstac there is a `remark` admonition available in `sphinx-proof` but they are pretty in your face styling. 

<img width="779" alt="Screenshot 2024-05-09 at 10 03 48 AM" src="https://github.com/QuantEcon/lecture-python-intro/assets/8263752/3890db17-b69c-492d-8a56-e3378f54cfa3">

I think it would be good to migrate to them and them update the styling perhaps?

Reply:
Yeah, +1 on that @mmcky .  These admonitions need simple styling.

Then definitions, theorems, etc. can have stronger styles, so they stand out.

Reply:
Thanks @jstac I have opened an issue  https://github.com/QuantEcon/lecture-python-intro/issues/442 to track